routers/web/org/projects.go

```diff
@@ -105,7 +107,17 @@ func Projects(ctx *context.Context) {
 	}
 
 	for _, project := range projects {
-		project.RenderedContent = templates.SanitizeHTML(project.Description) // FIXME: is it right? why not render?
+		project.RenderedContent, err = markdown.RenderString(&markup.RenderContext{
```

Comment:
I think we should use `desc, err := markup.RenderDescriptionHTM(` but not render all the markdown here since all of other places like repository description, org description did so.

Reply:
> @lunny these places already support Markdown, only their editor indicates otherwise as far as I understood
@delvh I don't think so, at least for `Project.Description`.